azurerm/internal/services/databoxedge/databox_edge_orders_resource.go

```diff
@@ -0,0 +1,586 @@
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"company_name": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeCompanyName,
+						},
+
+						"name": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeContactName,
+						},
+
+						"emails": {
+							Type:     schema.TypeSet,
+							Required: true,
+							ForceNew: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+
+						"phone_number": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgePhoneNumber,
+						},
+					},
+				},
+			},
+
+			"current_status": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"comments": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"additional_order_details": {
+							Type:     schema.TypeMap,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+
+						"update_date_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"shipping_address": {
+				Type:     schema.TypeList,
+				Required: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"address_line1": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeStreetAddress,
+						},
+
+						"city": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeCity,
+						},
+
+						"country": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeCountry,
+						},
+
+						"state": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeState,
+						},
+
+						"postal_code": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgePostalCode,
+						},
+
+						"address_line2": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeStreetAddress,
+						},
+
+						"address_line3": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ForceNew:     true,
+							ValidateFunc: validate.DataboxEdgeStreetAddress,
+						},
+					},
+				},
+			},
+
+			"delivery_tracking_info": {
```

Comment:
```suggestion
			"shipment": {
```

Reply:
How about we compromise, `shipping_info`? 🙂

Reply:
how about `shipment_info` or `shipment_tracking` - shipping seems to be the wrong verb for this?

Reply:
Fixed.